Taskfile.yaml

```diff
@@ -0,0 +1,94 @@
+# https://taskfile.dev
+
+version: '3'
+
+dotenv:
+  - '.env'
+
+vars:
+  VERSION:
+    sh: cat node/main.go | grep "Quilibrium Node - v" |  grep -Eo '[0-9]+\.[0-9]+\.[0-9]+'
```

Comment:
this is extracting the version from `node/main.go`, the version shows up in `db_console.go` as well, and ideally should be returned by `GetNodeInfo` I think.

@CassOnMars I can send a PR to extract the version as a constant in either a new file or one of the existing one. And add it to `GetNodeInfo` as well. Let me know.

Reply:
Seems reasonable, I'm supportive :+1: